.npmignore

```diff
@@ -12,6 +12,7 @@
 /.env*
 /.eslintignore
 /.eslintrc.js
+/.git/
```

Comment:
this is ignored by default, no need to add:
https://docs.npmjs.com/using-npm/developers.html#keeping-files-out-of-your-package

Reply:
It was added to the addon blueprint in 3.13: https://github.com/ember-cli/ember-addon-output/commit/7aea9e08d5711cb79a8eb86ef0281f9de63b621f#diff-0fd4ef892d9d4990033701887c2f9bcc

It was implemented in this commit: https://github.com/ember-cli/ember-cli/commit/14ba465649362a7f5903fad958de97a520ea900e The commit message says that at least in one case the `/.git/` directory was not ignored by default. But it seemed to be unclear why.